po/uk.po

```diff
@@ -17529,7 +17540,7 @@ msgid ""
 "There is work in progress on an `async` version of `embedded-hal`, but it "
 "isn't stable yet."
 msgstr ""
-"Виконується робота над асинхронною версією `embedded-hal`, але вона ще не є "
+"Виконується робота над `async` версією `embedded-hal`, але вона ще не є "
```

Comment:
Previous version seems less confusing for me. The crate got named `embedded-hal-async`. Anyway, probably no actions required since the source text was [updated](https://github.com/google/comprehensive-rust/commit/45aa43f4063fca460ba6c26ea02c869fe0cdf887) with:
> ```[`embedded-hal-async`](https://crates.io/crates/embedded-hal-async) provides async versions of the traits.```

`messages.pot` should be updated to not to do the work twice.

Reply:
Thanks, yeah, `messages.pot` will be updated (may intention was to do that at the end, there are quite a few changes).